tests/README.md

```diff
@@ -0,0 +1,40 @@
+# Intel® Media SDK Open Source TEst Driver
```

Comment:
How about to simplify the title? I think "Test Driver" without capital E will be much better...

Reply:
Done